core/trino-main/src/main/java/io/trino/sql/planner/optimizations/UnaliasSymbolReferences.java

```diff
@@ -647,6 +648,23 @@ public PlanAndMappings visitTableExecute(TableExecuteNode node, UnaliasContext c
             return new PlanAndMappings(rewrittenTableExecute, mapping);
         }
 
+        @Override
+        public PlanAndMappings visitSimpleTableExecuteNode(SimpleTableExecuteNode node, UnaliasContext context)
+        {
+            Map<Symbol, Symbol> mapping = new HashMap<>(context.getCorrelationMapping());
+            SymbolMapper mapper = symbolMapper(mapping);
+
+            Symbol newOutput = mapper.map(node.getOutput());
```

Comment:
Seems incorrect, as the `node.getOutput()` is produced by the  `SimpleTableExecuteNode` itself.
cc @kasiafi

Reply:
pretty much same thing is done here https://github.com/trinodb/trino/blob/a57c6c769101e7aeb91efd61ab8130f30516aae5/core/trino-main/src/main/java/io/trino/sql/planner/optimizations/UnaliasSymbolReferences.java#L584-L594

Reply:
Doesn't constitute a proof yet.

cc @martint @kasiafi

Reply:
yep it may not be 100% correct but it means it works at least good enough that nobody noticed so far ;) 
but of course I can try to change that but I am not sure how

Reply:
It is both correct and necessary to map the output symbol, even if it does not exist in the upstream plan, and is only introduced in the `SimpleTableExecuteNode`. The same applies to the Delete's output, and all the symbols introduced by nodes, e.g. unique id, row number,...

Please note that Unalias is used for two purposes:
1. to unify the Symbol's name throughout its path in the plan tree -- so that Explain is easier to read. In this case, the output symbol in question will remain unaffected in the `SimpleTableExecuteNode`, and all further references to this symbol will be renamed to the original name.
2. to reallocate all symbols in the plan for the purpose of a recursive query. In this case, the output symbol will be exchanged for a new symbol, and the new name will be used consistently in the plan. I don't think that execute will ever be part of a recursive query though.